tools/wptrunner/wptrunner/wptrunner.py

```diff
@@ -139,166 +140,180 @@ def get_pause_after_test(test_loader, **kwargs):
 def run_tests(config, test_paths, product, **kwargs):
     """Set up the test environment, load the list of tests to be executed, and
     invoke the remainder of the code to execute tests"""
-    with capture.CaptureIO(logger, not kwargs["no_capture_stdio"]):
```

Comment:
I have not particularly carefully checked the diff here as its a bit of a pain to review. I am assuming its just an indent + adding some instrumentation calls.